streamalert_cli/_infrastructure/modules/tf_cloudwatch_events/cross_account/README.md

```diff
@@ -0,0 +1,39 @@
+# StreamAlert CloudWatch Events Cross Account Terraform Module
+Configure the necessary resources to allow for cross account CloudWatch Events via EventBridge Events Bus
+
+## Components
+* Configures CloudWatch Event Permissions to allow external accounts or organizations to send events to the main account
+
+## Example
+```hcl
+module "cloudwatch_events_cross_account" {
+  source         = "./modules/tf_cloudwatch_events/cross_account"
+  accounts       = ["123456789012"]
+  organizations  = ["o-aabbccddee"]
```

Comment:
The organizations is a map, based on previous documentation.

Reply:
how it's defined in the conf file is different than how it's provided to the terraform module. see here: https://github.com/airbnb/streamalert/pull/1276/files#diff-1507d92d3a573dddd7e0a3dd89265421R79-R91

Reply:
it's defined in conf as a map, of account <> regions.. but is fed to terraform differently